src/main/java/bio/terra/cli/service/SamService.java

```diff
@@ -530,6 +382,21 @@ private void closeConnectionPool() {
         statusCode, "Error calling /api/google/v1/user/petServiceAccount/{project}/key endpoint");
   }
 
+  /**
+   * Utility method that checks if an exception thrown by the SAM client matches the given HTTP
+   * status code.
+   *
+   * @param ex exception to test
+   * @return true if the exception status code matches
+   */
+  private static boolean isStatusCode(Exception ex, int statusCode) {
```

Comment:
maybe name it `isHttpStatusCode` so it's clear it only works on a subset of exception statuses.

Reply:
done